.github/workflows/on_dispatch_deploy_test.yml

```diff
@@ -0,0 +1,12 @@
+name: deploy to testing
+
+on:
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    needs: [build]
+    uses: ./.github/workflows/reusable_deploy.yml
```

Comment:
I had the same thought. When I tried that, GitHub complained that everything had to be in the `workflows` directory. I dunno if I did something wrong, but it just didn't work. 🙁

Reply:
hmmm i am 99% sure i've seen this pattern, let me see if i can find one!

Reply:
I'm going to read more about composite action steps today. It looks like those may be able to solve this exact issue.

Reply:
Ahhh, it looks like what i've seen is the shared components pulled out into `actions`, which are composite run steps:

Here is an example of that setup: https://github.com/OHS-Hosting-Infrastructure/complaint-tracker/tree/main/.github/actions
 
I think this example is pretty clean, but I'd also understand if you didn't want to totally rearchitect this (not sure how much effort it would take) so up to you. also fine if you wanted to open an issue to possible reconsider this once everything is working!

Reply:
Let's make an issue for it. It doesn't look like a *lot* of work, but I'm also keen to get this one wrapped up.

Reply:
💫 iteration 💫